api/src/main/java/org/openmrs/User.java

```diff
@@ -147,9 +147,7 @@ public boolean hasRole(String r, boolean ignoreSuperUser) {
 		
 		Set<Role> tmproles = getAllRoles();
 		
-		if (log.isDebugEnabled()) {
-			log.debug("User #" + userId + " has roles: " + tmproles);
-		}
+			log.debug("User # {} has roles: {}", userId, tmproles);
```

Comment:
please remove the space between `#` and the `{}` so instead of `# {}` it should be `#{}`